torch_xla/distributed/xrt_init.py

```diff
@@ -178,7 +178,7 @@ def init_xrt_context(master_addr=None, master_port=None, store=None):
 
   # Call this in the actual test case, to work with torch/xla workers
   rank = int(os.environ['RANK'])
-  local_rank = int(os.environ['LOCAL_RANK'])
+  local_rank = int(os.environ[xenv.XLA_LOCAL_RANK])
```

Comment:
Will this break `torchrun`? Code that depends on torchrun will look for `LOCAL_RANK`

Reply:
Good point, updated the torchrun tests and variables.